package.json

```diff
@@ -4,10 +4,19 @@
   "description": "[Techtonica](https://techtonica.org) is a nonprofit that provides free tech training with stipends and job placement to women and non-binary adults with low incomes.",
   "main": "app.js",
   "scripts": {
+<<<<<<< HEAD
```

Comment:
@MariaKhantech I haven't reviewed the rest yet but I wanted to let you know these conflict markers ended up being committed, which will cause some problems, since the JSON is no longer valid. When I ran `npm install` it actually specifically warned me there were conflict markers in the file.

Before addressing any other comments/reviews can you go through all the files and make sure none of these markers are still there? Thanks!

If you're using VSCode (or Codespaces), there should be a UI to highlight these and let you decide what to do with them. Sometimes with conflicts, you take only one side of the conflict, but for these I think you'll need to combine them manually. 

If you need any help, feel free to ping an OSD mentor or me in the OSD Slack and we can try to help.